resource_booking/models/resource_booking.py

```diff
@@ -63,6 +63,7 @@ class ResourceBooking(models.Model):
         ),
     )
     name = fields.Char(index=True, help="Leave empty to autogenerate a booking name.")
+    description = fields.Text("Description")
```

Comment:
Why not HTML to have rich text format?

Reply:
The client didn't specify, so it's ok for me to change to html.

Reply:
done.